WorkflowFunctionAdminBundle/Transformer/WorkflowFunctionTransformer.php

```diff
@@ -33,7 +37,7 @@ public function __construct(TranslationChoiceManager $translationChoiceManager,
      */
     public function transform($mixed)
     {
-        $facade = new WorkflowFunctionFacade();
+        $facade = $this->newFacade();
```

Comment:
@Thiblef : Can you throw an exception ``if (!$facade instanceof FacadeInterface)``